editor/src/components/canvas/controls/select-mode/remix-scene-label.tsx

```diff
@@ -243,14 +255,13 @@ const RemixSceneLabel = React.memo<RemixSceneLabelProps>((props) => {
         className='roleComponentName'
         style={{
           pointerEvents: labelSelectable ? 'initial' : 'none',
-          color: isIndexRoute ? colorTheme.aqua.value : colorTheme.textColor.value,
+          color: colorTheme.aqua.value,
           position: 'absolute',
           left: frame.x,
           bottom: -frame.y + offsetY,
           width: frame.width,
-          paddingLeft: offsetX,
-          paddingTop: paddingY,
-          paddingBottom: paddingY,
+          height: scaledLineHeight * 2,
+          padding: '10px 20px',
```

Comment:
I think these need to be zoom-safe, so I'd either restore the previous code or adapt it